CodeMaid/Helpers/CodeItemTypeComparer.cs

```diff
@@ -118,16 +119,20 @@ private static int CalculateAccessOffset(BaseCodeItem codeItem)
             var codeItemElement = codeItem as BaseCodeItemElement;
             if (codeItemElement == null) return 0;
 
-            switch (codeItemElement.Access)
-            {
-                case vsCMAccess.vsCMAccessPublic: return 1;
-                case vsCMAccess.vsCMAccessAssemblyOrFamily: return 2;
-                case vsCMAccess.vsCMAccessProject: return 3;
-                case vsCMAccess.vsCMAccessProjectOrProtected: return 4;
-                case vsCMAccess.vsCMAccessProtected: return 5;
-                case vsCMAccess.vsCMAccessPrivate: return 6;
-                default: return 0;
-            }
+            var itemsOrder = new[] {
+                vsCMAccess.vsCMAccessPublic,
+                vsCMAccess.vsCMAccessAssemblyOrFamily,
+                vsCMAccess.vsCMAccessProject,
+                vsCMAccess.vsCMAccessProjectOrProtected,
+                vsCMAccess.vsCMAccessProtected,
+                vsCMAccess.vsCMAccessPrivate
+                };
+            if (!Settings.Default.Reorganizing_RevertAccessLevel)
```

Comment:
Revert isn't a clear verb to me (means go back to an original state).  I think you mean Reverse access level.  In that case though, wouldn't this be a positive case where only if the flag was set would the reversal happen?